core/utilities.lua

```diff
@@ -427,6 +427,63 @@ utilities.utf8_to_utf16le = function (str)
   return ustr
 end
 
+local icu = require("justenoughicu")
```

Comment:
Now that ICU is just a requirement rather than an optional add-on, I think we should probably load one instance of it as a global or into SU or somewhere rather than having to load it in every module separately. Thoughts?

Reply:
My understanding is that it doesn't matter. Lua caches loaded packages (into a table called `packages.loaded`) so you're not really "loading" it again. I think this way is actually more declarative and easy to follow.

Reply:
Savvy.